src/lib/markbind/src/preprocessors/componentPreprocessor.js

```diff
@@ -318,6 +315,16 @@ function _preprocessVariables() {
   return utils.createEmptyNode();
 }
 
+function _preprocessImports(node, parser) {
+  if (node.attribs.from) {
+    parser.staticIncludeSrc.push({
```

Comment:
previously this is done inside the variable processing methods, hence `variablePreprocessor` needs to know of `parser`. I moved it here to remove that dependency